src/core/public/chrome/ui/header/collapsible_nav_group_enabled.scss

```diff
@@ -1,33 +1,57 @@
 .context-nav-wrapper {
   border: none !important;
+  border-top-right-radius: $euiSizeL;
+  border-bottom-right-radius: $euiSizeL;
+  background-color: $euiColorLightShade;
```

Comment:
<img width="152" alt="image" src="https://github.com/user-attachments/assets/532c0547-0285-46a0-9576-f809ab5af0d5">
We should perhaps add `overflow: hidden` here, otherwise it looks like this

Reply:
Thanks, fixed.